src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs

```diff
@@ -429,6 +429,42 @@ private int DownFocusableViewId
             }
         }
 
+        private int ClockwiseFocusableViewId
+        {
+            get
+            {
+                int returnValue = 0;
```

Comment:
As I understand, "`ClockwiseFocusableView` is not exist, then `ClockwiseForcusableViewId` shoudl be -1".

In that case, should `returnValue = -1;` is more right way?